pmpro-set-expiration-dates.php

```diff
@@ -30,7 +30,11 @@ function pmprosed_pmpro_membership_level_after_other_settings()
         $set_expiration_date = "";
     ?>
     <h3 class="topborder"><?php _e( 'Set Expiration Date', 'pmpro-set-expiration-dates' ); ?></h3>
-    <p><?php _e( 'To have this level expire on a specific date, enter it below in YYYY-MM-DD format. <strong>Note:</strong> You must also set an expiration date above (e.g. 1 Year) which will be overwritten by the value below.', 'pmpro-set-expiration-dates' ); ?></p>
+    <p><?php printf( 
+        __( 'To have this level expire on a specific date, enter it below in YYYY-MM-DD format. %1$sNote:%2$s You must also set an expiration date above (e.g. 1 Year) which will be overwritten by the value below.', 'pmpro-set-expiration-dates' ),

```

Comment:
I think concatenating is better here.

Something like this:

`echo __( 'To have this level expire on a specific date, enter it below in YYYY-MM-DD format., 'pmpro-set-expiration-dates') . '<strong>' . __( 'Note', 'pmpro-set-expiration-dates' ) . ':</strong>' . __( 'You must also set an expiration date above (e.g. 1 Year) which will be overwritten by the value below.', 'pmpro-set-expiration-dates' ),`